presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -85,6 +95,6 @@ public void testNativeExecutionShuffleManager()
         BypassMergeSortShuffleHandle<?, ?> bypassMergeSortShuffleHandle = (BypassMergeSortShuffleHandle<?, ?>) shuffleHandle.get();
         int shuffleId = shuffleHandle.get().shuffleId();
         assertEquals(0, shuffleId);
-        assertEquals(shuffleManager.getNumOfPartitions(shuffleId), bypassMergeSortShuffleHandle.numMaps());
+        assertEquals(shuffleManager.getNumOfPartitions(shuffleId), bypassMergeSortShuffleHandle.dependency().partitioner().numPartitions());
```

Comment:
Are these assertions specific to a query or generic? Any reason not to move these into 'assertQuery'?